pages/pokemons/[id].vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div class="pokemon">
+    <h1>Pokemon Details</h1>
+
+    <div v-if="pokemon" class="pokemon-detail">
+      <img :src="pokemon.imageUrl" class="pokemon-image" :alt="pokemon.name" />
+      <h2 class="pokemon-name">{{ pokemon.name }}</h2>
+      <span class="pokemon-id">ID: {{ pokemon.id }}</span>
+      <span class="pokemon-height">Height: {{ pokemon.height }}</span>
+      <span class="pokemon-weight">Weight: {{ pokemon.weight }}</span>
+      <FavouriteButton :pokemon="pokemon" />
+    </div>
+
+    <div v-if="isLoading" class="loading">Loading...</div>
+    <div v-if="error" class="error">Failed to fetch Pokemon details</div>
+
+    <button class="back-button" @click="goBack">Back</button>
```

Comment:
is this semantically correct? 
Would you be able to open new tab when clicking on this button?